### ISSUE TITLE
Asking for an example as I don't get anything but login

### ISSUE THREAD

**Issue by moisty70** [lorenzh/fut-api#22]
First thanks for the project.

I am trying to get some results with no success. I can just login.

getStatus, getTradepile, search... nothing seems to send a response as shown in the readme file.

Could someone send an example of a search, for example silver players with position ST.

I suppose its something like this:
apiClient.search({type: "player", lev: "silver", pos: "ST" }, function(error, response){ });

but... ¿How I get to the players list?

**Comment by pjar** [comment]
Once you successfully logged in you can do something like:

```
apiClient.search({num: 48, type: "player", lev: "silver", pos: "ST"}, function(error, response){ 
    if (!response.auctionInfo) {
      // Return full response to see why there are no auctions
      console.log(response);
      return;
    } else {
       // Return auction entries when it's successful
       console.log(response.auctionInfo);
    }
};
```

**Comment by moisty70** [comment]
if(!response.auctionInfo) is true. How can It be?

**Comment by moisty70** [comment]
ok. I got it. 
btw. num:48 is the pagesize of the items query.

**Comment by moisty70** [comment]
thanks. got it working. 
i will wake up in the morning with 200000 more coins very soon.

next test I will do is to put all this in a loop to check every 30 seconds. can EA ban a user for doing this?

**Comment by aphrs** [comment]
What's your method ? Do you only place bid ?

**Comment by moisty70** [comment]
my method for players needed for SBC will be place bid.
for consumables like healing or squad training will be buy now. buy on low price in the weekdays higher sold on fut champions weekend

**Comment by aphrs** [comment]
Well i'm interested on buy now method, I don't see it on the API but only this one to place bid : `apiClient.placeBid(tradeId, coins, function(error, response){ });` 
How do you buy now ?

**Comment by moisty70** [comment]
I bought Chelsea's GK Begovic yesterday setting coins parameter equal to the buy now price of the tradeId using a placeBid call. So it is a "buyNow" if you use the correct amount.

**Comment by yhanssens** [comment]
Did you manage to get any results @moisty70 ? I seem to have the same problem. The response code I get is 494. A quick google search gives me the following on that:  494 Request Header Too Large.
Not sure what to do now :)

**Comment by moisty70** [comment]
@joertler yes I've done a loop to buy players at my desired price and works, Post your code here to check it.
If you want I can post mine. Not in the same PC now.

**Comment by yhanssens** [comment]
Already figured it out :) Thanks anyway! Currently working on some trade tactics. Did you find a rewarding one?
